File: Java/1008 - Salário.java
import java.io.IOException;
import java.util.*;

public class Main{
    public static void main(String[] args){
        int numFunc;
        double valorHora, valorTotal, horasTrabalhadas;
        
        Scanner valor = new Scanner (System.in);
        numFunc = valor.nextInt();
        valorHora = valor.nextDouble();
        horasTrabalhadas = valor.nextDouble();
        
        valorTotal = valorHora * horasTrabalhadas;
        
        System.out.println("NUMBER = " + numFunc); 
        System.out.println(String.format("SALARY = U$ %.2f", valorTotal));
    }
}

File: C#/1008 - Salário.cs
using System;

namespace Salario{
    class Program{
        static void Main(string [] args){
            int numFunc, horasTrabalhadas;
            double valorHora, salario;
            
            numFunc = Convert.ToInt32(Console.ReadLine());
            horasTrabalhadas = Convert.ToInt32(Console.ReadLine());
            valorHora = Convert.ToDouble(Console.ReadLine());
            
            salario = valorHora * horasTrabalhadas;
            
            Console.WriteLine("NUMBER = " + numFunc);
            Console.WriteLine(string.Format($"SALARY = U$ {salario.ToString("0.00")}"));
        }
    }
}
